docs/joss_paper/paper.md

```diff
@@ -0,0 +1,104 @@
+contributions stated in [@adye2].
+
+
+The primary purpose of PyUnfold is to provide an unfolding toolkit for members of all 
+scientific disciplines in an easy-to-use package.
+For example, unfolding methods are commonly used in the analysis pipeline of the 
+high-energy physics (HEP) community.
+However, the deconvolution packages used in HEP maintain a strong dependence on the 
+ROOT data analysis framework [@root], which is almost exclusively used in HEP.
+Instead, PyUnfold is built on top of the Python scientific computing stack (i.e. NumPy,
+SciPy, and pandas), thus broadening its scope to a general scientific audience.
+
+
+Indeed, the method itself is data-agnostic, referring to the measurement process
+as the smearing of a set of true causes into a set of detectable effects.
+For example one could define as causes the true energy of a particle and the effects
+the measured energy of that particle in a detector.
+Another example might be a set of diseases (causes) and possible clinical symptoms (effects).
+So long as it is possible to encode estimable resolutions and biases connecting causes to 
+effects in a binned response matrix, one can perform a deconvolution with PyUnfold. 
+
+
+PyUnfold has been designed to be both easy to use for first-time users as well as 
+flexible enough for fine-tuning an analysis.
+Its main strength is its simplicity, requiring a single line to run via the ``iterative_unfold`` 
+method, taking as input the user provided distributions and other runtime parameters.
+This also facilitates examining the effects of varying optional parameters on the final 
+unfolded distribution, allowing users to seamlessly test the robustness of results.
+
+PyUnfold provides support for the following:
+
+- Custom, user defined initial prior probability distributions, the default being 
+the uniform prior. The non-informative Jeffreys' prior [@jeffreys] is accessible 
+as a utility function. 
+
+- Unfolding stopping criteria based on test statistic calculations comparing unfolded
+distributions from one iteration to the next. These include 
+Kolmogorov-Smirnov[@kolmogorov][@smirnov], $\chi^2$, relative difference, 
+and Bayes factor [@pfendner] tests.
+
+- Tunable spline regularization as a means of ensuring that unfolded distributions do not 
+suffer from growing fluctuations potentially arising from the finite binning of the
+response matrix.
+
+- Option to choose between Poisson or multinomial forms of the covariance matrices
+for both the data and response contributions to the uncertainty calculation.
+
+- Multivariate unfolding via definitions of subsets of causes, which are regularized 
+in their respective blocks or groups.
+
+
+Further mathematical details regarding the iterative unfolding procedure, including complete 
+derivations of the statistical and systematic uncertainty propagation can be found in the 
+\LaTeX \ documentation.
```

Comment:
I think this should just say "online documentation" (we can even link to https://jrbourbeau.github.io/pyunfold/). Most software projects don't have accompanying LaTeX writeups, so this might be confusing to some people. Also, the corresponding section in the PyUnfold documentation has a link to the LaTeX document.